README.md

```diff
@@ -59,6 +59,5 @@ Promise.race([
 
 ## Unhandled Rejections
 
-In node lie emits `unhandledRejection` events when promises are not handled in
-line with how [iojs does so](https://iojs.org/api/process.html#process_event_unhandledrejection)
-meaning it can act as promise shim in node as well as the browser.
+
+lie emits an `unhandledRejection` event when a rejected promise isn't caught, in line with [how iojs does it](https://iojs.org/api/process.html#process_event_unhandledrejection). This allows it to act as a promise shim in both node and the browser.
```

Comment:
I would specify that lie only emits an `unhandledRejection` in the node. It does not emit this event in the browser (assuming I am reading the code correctly)